test/unit-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as assert from "assert";
+import * as functions from "./../src/functions/functions";
+import * as mocha from "mocha";
+
+describe("Test Excel Custom Functions", function () {
+    describe("Test add function", function () {
+        it(`Add function should return expected value`, async function () {
+            assert.equal(functions.add(5, 6), 11);
+        });
+    });
+    describe("Test logMessage function", function () {
+        it(`LogMessage function should return expected value`, async function () {
+            const message: string = "Hello World!"
+            assert.equal(functions.logMessage(message), message);
+        });
+    });
+    describe("Test currentTime function", function () {
+        it(`CurrentTime function should return time value`, async function () {
+            const time = functions.currentTime();
+            assert.equal(time.includes("AM") || time.includes("PM"), true);
```

Comment:
This isn't the best test case as the format will differ by locale. It would fail on locales with a 24 clock. I would just confirm that it returns a string.